IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -526,7 +526,7 @@ use Digest::MD5 ('md5_hex');
 use v5.10;
 
 use Readonly;
-Readonly my $I7_REGEX    => qr/\.z\d$|\.[gz]?blorb$|\.ulx$/i;
+Readonly my $I7_REGEX    => qr/\.z\d$|\.[gz]?blorb$|\.ulx$|\.gam$|\.t3$/i;
```

Comment:
I think this regex is not working properly, and it's catching other files. Does the `qr` mode mean that the period isn't being escaped properly?